### PR TITLE
Sql ssasmd ssrs ad two vm

### PR DESCRIPTION
**Pull request by markvsql**
### Contributing guide

https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md
### Changelog

*
*
*
*
*
### Description of the change

This pull requests is for the initial addition of this template to the repository master branch.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@markvsql__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "rootEnvironmentName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the environment. Use all lowercase as this will be the basis for many resource names."
+      }
+    },
+    "location": {
```
Comment (singhkay) [inline sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json]: remove and use resourceGroup().location wherever needed

File: sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+        "description": "The size of the SQL VM Created"
+      },
+      "defaultValue": "Standard_DS11"
+    },
+    "sqlServerServiceAccountUserName": {
+      "type": "string",
+      "metadata": {
+        "description": "The SQL Server Service account name"
+      },
+      "defaultValue": "sqlservice"
```
Comment (singhkay) [inline sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json]: remove default value as a security best practice

File: sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,626 @@
...
+      "metadata": {
+        "description": "The SQL Server Service account password"
+      }
+    },
+    "sqlServerfOffer": {
+      "type": "string",
+      "metadata": {
+        "description": "The offer for the SQL Server VM"
+      },
+      "defaultValue": "SQL2014SP1-WS2012R2"
```
Comment (singhkay) [inline sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json]: How about 2016 ?

File: sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,626 @@
...
+    }
+  },
+  "variables": {
+    "armfileLocation": "[parameters('baseURL')]",
+    "armfiledscLocation": "[concat(parameters('baseURL'),'/dsc')]",
+    "adVMName": "[concat(parameters('rootEnvironmentName'),'-ad')]",
+    "sqlVMName": "[concat(parameters('rootEnvironmentName'),'-sql')]",
+    "vNetName": "[concat('vn',parameters('rootEnvironmentName'))]",
+    "domainName": "[concat(parameters('rootEnvironmentName'),'.local')]",
+    "storageAccountNamePrefix": "[concat('st',parameters('rootEnvironmentName'))]",
```
Comment (singhkay) [inline sql-ssasmd-ssrs-ad-two-vm/azuredeploy.json]: generate a storage account name rather than using any user input. See #944

**Comment by gatneil** [comment]
No response; closing

**Comment by markvsql** [comment]
Sorry I haven’t made it back to this. Thanks.



Mark Vaillancourt | Data Solution Architect | Microsoft Corporation | US North Central District
Office: (952)656-2284 | Cell: (612)247-3858
[MSFT_logo_Gray DE sized SIG1.png]

**Pull request closed without merging**